conda/cli/main_search.py

```diff
@@ -8,6 +8,8 @@
 from collections import defaultdict
 from datetime import datetime, timezone
 
+from ..models.records import PackageRecord
```

Comment:
This will be too expensive of an import hit at runtime, encapsulate in a type-checking block instead:

```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from ..models.records import PackageRecord
```